SQL/0000-00-02-Menus.sql

```diff
@@ -35,9 +35,10 @@ INSERT INTO LorisMenu (Label, OrderNumber) VALUES
      ('Candidate', 1),
      ('Clinical', 2),
      ('Imaging', 3),
-     ('Reports', 4),
-     ('Tools', 5),
-     ('Admin', 6);
+     ('Electrophysiology', 4)
```

Comment:
Should this be under Imaging?

Reply:
Actually, this is a new modality altogether not really considered as imaging but as electrophysiology (at least this is my understanding, @christinerogers what do you think?)

Reply:
Cool, just checking. I definitely don't know.

Reply:
You're both right, and I remember discussing this in 2018. 
You're definitely correct @cmadjar that it's not actually imaging, and that's why in 2018 we decided it gets its own menu.
My personal feeling is it should be under Imaging (👍 @johnsaigle ) because: 
(a) it's close enough to imaging, in terms of the context of the researchers, and compared to the testing/visit workflows used for behavioural/genomic...  
Plus, Loris Menu "real estate" in 2 senses:
(b) having a menu with just one item, as well as:
(c) having a super-wide menu item -- this genuinely messes up the Loris menu look at med-large screens and smaller